newspaper/images.py

```diff
@@ -167,6 +171,52 @@ def fetch_image_dimension(url, useragent, referer=None, retries=1):
     return fetch_url(url, useragent, referer, retries, dimension=True)
 
 
+def valid_image_url(url):
+
+    url = clean_url(url)
+    if not url.startswith(('http://', 'https://')):
+        return False
+    path = urlparse(url).path
+
+    # input url is not in valid form (scheme, netloc, tld)
```

Comment:
Same as above, we don't know the expectation of this function, the input url may be relative, e.g. `/images/content/red_car.jpg`